tests/scripts/visual-coverage.sh

```diff
@@ -1,16 +1,24 @@
 #!/bin/bash
 
-# start Storybook in the same shell
-. ./tests/scripts/start-storybook.sh
+#check for existing process on port:6006
+process=`lsof -i :6006`
+if [[ -z $process ]]
+then
+  echo "port:6006 available - starting storybook"
+  # start Storybook in the same shell
+  . ./tests/scripts/start-storybook.sh
 
-# generate coverage from storybook test-runner
-test-storybook --coverage
+  # generate coverage from storybook test-runner
+  test-storybook --coverage
 
-# kill the locally served Storybook
-kill -9 $JOB_ID
+  # kill the locally served Storybook
+  npx kill-port 6006
```

Comment:
instead of killing a hardcoded port of 6006, can we instead of start-storybook.sh return the job id (process id? I don't know the terminology) of the storybook instance, and kill that process here instead? that way we don't have a hardcoded port of 6006